locale/fa_IR/emailTemplates.xml

```diff
@@ -0,0 +1,1105 @@
+<br />
+حساب کاربری شما برای مجله {$contextName}, ایجاد شد. قبل از استفاده می بایست آن را تایید کنید. برای انجام این کار فقط کافی است که بر روی لینک زیر کلیک کنید:<br />
+<br />
+<a href="{$activateUrl}">{$activateUrl}</a><br />
+<br />
+با تشکر<br />
+{$principalContactSignature}]]></body>
+		<description>This email is sent to a newly registered user to welcome them to the system and provide them with a record of their username and password.</description>
+	</email_text>
+	<email_text key="REVIEWER_REGISTER">
+		<subject>ثبت نام به عنوان کارشناس {$contextName}</subject>
+		<body><![CDATA[با توجه با تجارب جنابعالی, {$contextName} فرصت را مغتنم شمرده و نام شما را در زمره کارشناسان مجله ثبت کرده است. لازم به تذکر است که صرف این عمل هیچگونه مسئولیت و تعهدی را متوجه جنابعالی نمی سازد و فقط ما را قادر می سازد تا جهت ارسال مقاله برای کارشناسی احتمالی توسط جنابعالی براحتی با شما تماس بگیریم. هنگامی که از شما برای انجام کارشناسی یک مقاله دعوت به عمل می آید شما مختار خواهید بود که پس از دیدن عنوان و چکیده مقاله دعوت را بپذیرید یا از انجام کارشناسی خودداری کنید. بعلاوه در هر زمان که اراده کنید میتوانید از ما بخواهید تا نام شما را از لیست کارشناسان مجله حذف کنیم.<br />
+<br />
+نام کاربری و رمز عبور شما برای سایت مجله که جهت انجام تمام امورات آنلاین با این مجله لازم است در زیر آمده است.<br />
+<br />
+نام کاربری: {$username}<br />
+رمز عبور: {$password}<br />
+<br />
+با تشکر,<br />
+{$principalContactSignature}]]></body>
+		<description>This email is sent to a newly registered reviewer to welcome them to the system and provide them with a record of their username and password.</description>
+	</email_text>
+	<email_text key="PUBLISH_NOTIFY">
+		<subject>شماره جدید چاپ شد</subject>
+		<body><![CDATA[خوانندگان گرامی:<br />
+<br />
+{$contextName}، آخرین شماره خود را به نشانی &amp;lt;<a href="{$contextUrl}">{$contextUrl}</a>&amp;gt; چاپ کرده است. از شما دعوت میشود که پس از ملاحظه فهرست عناوین از وب سایت مجله حاوی مقاله ها و موضوع های مورد علاقه بازدید کنید.<br />
+<br />
+از توجه همیشگی مستمر شما نسبت به این مجله متشکریم.<br />
+{$editorialContactSignature}]]></body>
+		<description>This email is sent to registered readers via the "Notify Users" link in the Editor's User Home. It notifies readers of a new issue and invites them to visit the journal at a supplied URL.</description>
+	</email_text>
+	<email_text key="LOCKSS_NEW_ARCHIVE">
+		<subject>درخواست بایگانی کردن {$contextName}:</subject>
+		<body><![CDATA[مسئول محترم کتابخانه دانشگاه:<br />
+{$contextName} &amp;lt;<a href="{$contextUrl}">{$contextUrl}</a>&amp;gt; ، عضو دانشکده شما می باشد &amp;lt;[name of member]&amp;gt;. این مجله خواستار ایجاد سیستم بایگانی LOCKSS با آن کتابخانه و کتابخانه های دیگر دانشگاهها می باشد.<br />
+<br />
+[Brief description of journal]<br />
+<br />
+برنامه LOCKSS تحت آدرس http://lockss.org که یک ابتکار بین المللی مربوط به ناشر/کتابخانه می باشد. این نرم افزار رایگان، حتی توسط یک رایانه شخصی نیز اجرا می گردد. کل سامانه آن لاین شده است و به نگهداری مختصری نیازمند است.<br />
```

Comment:
It looks like this is incompletely translated -- is that intentional?

Reply:
all section check and the untranslated ones has been translated. only the descriptions translation is incomplete